ui-spacetimechart/src/types.ts

```diff
@@ -0,0 +1,6 @@
+export type WorkSchedule = {
+  type: 'TRACK' | 'CATENARY';
```

Comment:
This seems unused still, but I reckon we'll want to use this for the tooltip in the future? Since it's well-defined I don't mind introducing it in this PR.

Reply:
alright, since it is also defined in editoast I think we can leave it